File: ManagerIngestTag/ManagerIngestTag/Models/IngestDetailModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace ManagerIngest.Models
{
    public class IngestDetailModel
    {
        public Guid IngestDeltailId { get; set; }
        public string DateSend { get; set; }
        public string DateReturn { get; set; }
        public string RecipientName { get; set; }
        public Guid RecipientId { get; set; }
        public string TakerName { get; set; }
        public Guid TakerId { get; set; }
        public Guid ticketIngestId { get; set; }
        public Guid IngestTagId { get; set; }
    }

    public class IngestDetailFull : IngestDetailModel
    {
        public IngestTagReturnModel IngestTag { get; set; }

    }
}

File: MangagerInsestTag.FE/src/app/shared/IngestDetail/ingest-detail.model.ts
import { Ingest } from "../Ingest/ingest.model";

export class IngestDetail {
    IngestDeltailId: string = '00000000-0000-0000-0000-000000000000';
    DateSend: string = '';
    DateReturn: string = '';
    RecipientName?: string = '';
    RecipientId: string = '';
    TakerName: string = '';
    TakerId: string = '';
    ticketIngestId: string = '';
    IngestTagId: string = '';
    IngestTag: Ingest = new Ingest();
}
